test/verify/check-networkmanager-basic

```diff
@@ -165,6 +171,11 @@ class TestNetworkingBasic(NetworkCase):
         assert_running()
         wait(lambda: m.execute("systemctl is-enabled NetworkManager"))
 
+        # remove the NM service file, test 'no-found' notification
+        self.restore_file('/usr/lib/systemd/system/NetworkManager.service')
+        m.execute("rm /usr/lib/systemd/system/NetworkManager.service && systemctl daemon-reload && systemctl stop NetworkManager")
```

Comment:
This [does not work on coreos](https://logs.cockpit-project.org/logs/pull-15266-20210210-120426-ffc90af8-fedora-coreos/log.html#317).

I'd really rather this would not mess around in /usr. `systemctl mask --runtime` ought to work for most cases, you may need to mask some additional sockets or so (@mvollmer mentioned that masking was not enough, but I don't know how). Or move this into a separate destructive test and skip it on fedora-coreos.

Reply:
> I'd really rather this would not mess around in /usr.

I am afraid we have to. We want to simulate a completely non-present NetworkManager.  One way would be to uninstall the package, but it seems that removing just the service file is enough.

> Or move this into a separate destructive test and skip it on fedora-coreos.

Skipping on fedora-coreos is appropriate.